tasks/updateconstants.js

```diff
@@ -36,7 +36,7 @@ const extraStrings = {
   DOTA_UNIT_TARGET_HERO: "Hero",
   DOTA_UNIT_TARGET_BASIC: "Basic",
   DOTA_UNIT_TARGET_BUILDING: "Building",
-  DOTA_UNIT_TARGET_TREE: "Tree"
+  DOTA_UNIT_TARGET_TREE: "Tree",
```

Comment:
I'd put the formatting only changes in a separate PR to make this easier to review